core/src/config.rs

```diff
@@ -103,3 +103,23 @@ fn admin_public_key_from(
 
     Ok(PublicKey::from(&SecretKey::from_bytes(&hash)?))
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+
+    #[test]
+    fn admin_public_key() {
+        let email: String = "pj@aa.pl".to_string();
+        let password: String =  "password".to_string();
+        let seed: Option<[u8; 32]> = Some([55; 32]);
```

Comment:
```suggestion
        let seed = [55u8; 32];
```

And then use `Some(seed)` below.

Reply:
What is the benefit of this?

Reply:
First, optionality is not part of semantics of that variable, if it was, the proper name for it would be something like `maybe_seed`.

Second, it's a concrete value and the only thing we wrap it into `Result` is `Config::new` calling conventions, i.e. something incidental to having that value in the first place. Given that this is a part of calling convention, rather than something meaningful in and of itself, it would make more sense to write `Config::new(..., Some(seed))`.

Reply:
👍